src/plone/testing/_z2_testbrowser.py

```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
 from cStringIO import StringIO
+from zope.testbrowser.testing import PublisherConnection
+from zope.testbrowser.testing import PublisherResponse
+
 import base64
 import mechanize
 import re
 import rfc822
 import sys
 import urllib
 import urllib2
-
-# Using a from-import here to avoid an AttributeError below when using
-# zope.testbrowser 4.x without zope.app.testing:
-from zope.testbrowser.testing import PublisherConnection, PublisherResponse
```

Comment:
It's quite late and I'm tired, so the tl&dr; version is that we can merge this pull request and prepare another one (would you be my guest? 😄 ) that removes this ``_z2_testbrowser.py`` completely?